test/Mono.Linker.Tests.Cases/DataFlow/XmlAnnotations.xml

```diff
@@ -13,51 +13,51 @@
           <argument>0</argument>
         </attribute>
         <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute">
-          <argument>PublicParameterlessConstructor</argument>
+          <argument type="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes">PublicParameterlessConstructor</argument>
         </attribute>
       </field>
       <method signature="System.Void TwoAnnotatedParameters(System.Type,System.Type)">
         <parameter name="type">
           <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute">
-            <argument>PublicParameterlessConstructor</argument>
+            <argument type="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes">PublicParameterlessConstructor</argument>
           </attribute>
         </parameter>
         <parameter name="type2">
           <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute">
-            <argument>PublicConstructors</argument>
+            <argument type="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes">PublicConstructors</argument>
           </attribute>
         </parameter>
       </method>
       <method signature="SpacesBetweenParametersWrongArgument (System.Type, System.Boolean)">
         <parameter name="type">
           <attribute fullname="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute">
-            <argument>NonValidArgument</argument>
+            <argument type="System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes">0</argument>
```

Comment:
We need to test that a warning is produced when someone passes something for which the constructor was not found. In this case, 0 means MemberTypes.None so it should be a valid entry

Reply:
See the added test in the PR, there are tests for invalid arguments which are errors that would be the same case here so the whole test fixture would fail. There is no validation for this test so not sure what it's supposed to check.

Reply:
Yeah, I see that now we have a error test file, seems fine to me to have the failing test there then